zuul.d/ansible-utils-jobs.yaml

```diff
@@ -1,69 +1,39 @@
 ---
 # ansible-collections/ansible.utils jobs
-
-- job:
-    name: ansible-test-integration-ansible-utils
-    abstract: true
-    parent: ansible-test-integration-base
-    required-projects:
-      - name: github.com/ansible/ansible
-      - name: github.com/ansible-collections/ansible.utils
-    vars:
-      ansible_collections_repo: github.com/ansible-collections/ansible.utils
-
-- job:
-    name: ansible-test-integration-ansible-utils-python27
-    parent: ansible-test-integration-ansible-utils
-    nodeset: controller-python27
-    vars:
-      ansible_test_python: 2.7
-
 - job:
-    name: ansible-test-integration-ansible-utils-python27-stable211
-    parent: ansible-test-integration-ansible-utils
-    nodeset: controller-python27
-    required-projects:
-      - name: github.com/ansible/ansible
-        override-checkout: stable-2.11
-    vars:
-      ansible_test_python: 2.7
+    name: network-ee-integration-tests-utils
+    parent: network-ee-integration-tests-openvswitch-openvswitch
 
 - job:
-    name: ansible-test-integration-ansible-utils-python27-stable29
-    parent: ansible-test-integration-ansible-utils
-    nodeset: controller-python27
-    required-projects:
-      - name: github.com/ansible/ansible
-        override-checkout: stable-2.9
-    vars:
-      ansible_test_python: 2.7
+    name: network-ee-integration-tests-utils
+    parent: network-ee-integration-tests
+    nodeset: openvswitch-2.9.0-python38
 
 - job:
-    name: ansible-test-integration-ansible-utils-python36
-    parent: ansible-test-integration-ansible-utils
-    nodeset: controller-python36
+    name: network-ee-integration-ansible-utils-python38
+    parent: network-ee-integration-tests-utils
+    nodeset: openvswitch-2.9.0-python38
     vars:
-      ansible_test_python: 3.6
+      collection_name: utils
+      collection_namespace: ansible
 
 - job:
-    name: ansible-test-integration-ansible-utils-python36-stable211
-    parent: ansible-test-integration-ansible-utils
-    nodeset: controller-python36
-    required-projects:
-      - name: github.com/ansible/ansible
-        override-checkout: stable-2.11
+    name: network-ee-integration-ansible-utils-python38-stable211
+    parent: network-ee-integration-tests-utils
+    nodeset: openvswitch-2.9.0-python38
     vars:
-      ansible_test_python: 3.6
+      test_ansible_python_interpreter: /usr/bin/python3.6
+      collection_name: utils
+      collection_namespace: ansible
```

Comment:
these are not required, we load them from the parent

Reply:
This is necessary otherwise it took openvswitch as collection name and openvswitch as namespace from parent job. check this logs https://dashboard.zuul.ansible.com/t/ansible/build/a5ceaec3100f4f37a700f9e95510ff55/log/job-output.txt its taking ovs ovs and running ovs integration tests if we can not explicitly specify it.

Reply:
understood, we can keep then